modules/install-creating-aws-local-zones-subnet.adoc

```diff
@@ -0,0 +1,77 @@
+:_content-type: PROCEDURE
+[id="install-creating-aws-local-zones-subnet"]
+=== Creating an AWS Local Zone subnet
+
+To create a Local Zone subnet, choose a zone group to enable and opt it in. Then, create the subnet.  
+
+.Procedure
+
+. To opt the zone group in, from a command line, enter the following command:
```

Comment:
Maybe you can use the term "Enable" instead of "opt ... in", as AWS are using how to do it:

> "To enable a Local Zone using the AWS CLI"

https://docs.aws.amazon.com/local-zones/latest/ug/getting-started.html